src/components/pages/Offer/domain/localProvider.js

```diff
@@ -21,3 +31,15 @@ export const isOfferFromStockProvider = (offer = null) => {
 export const isAllocineOffer = (offer = null) => {
   return doesLastProviderExist(offer) ? offer.lastProvider.name.toLowerCase() === 'allociné' : false
 }
+
+export const isFieldReadOnlyForSynchronizedOffer = (fieldName, isAllocineOffer) => {
```

Comment:
```suggestion
export const isFieldReadOnlyForSynchronizedOffer = (fieldName, provider) => {
```
Je n'aime pas trop le flag booléen passé en entrée
Autant laisser cette méthode faire la logique de différencier en fonction du provider